autom4te: cannot open configure: Permission denied

```
$ git lg | head -n1
* commit c3bdc9482cad66b0e233717eaa8165265fdc2342 (HEAD -> master, origin/master)
$ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.4.0 (2020-03-21)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.4) pkg> build GAP
   Building GAP → `~/.julia/dev/GAP/deps/build.log`
Downloading artifact: gap-4-11-0
┌ Error: Error building `GAP`, showing the last 100 of log:
│ checking util.h usability... yes
│ checking util.h presence... yes
│ checking for util.h... yes
│ checking pty.h usability... no
│ checking pty.h presence... no
│ checking for pty.h... no
│ checking libutil.h usability... no
│ checking libutil.h presence... no
│ checking for libutil.h... no
│ checking signal.h usability... yes
│ checking signal.h presence... yes
│ checking for signal.h... yes
│ checking for select... yes
│ checking spawn.h usability... yes
│ checking spawn.h presence... yes
│ checking for spawn.h... yes
│ checking for sys/wait.h that is POSIX.1 compatible... yes
│ checking for pid_t... yes
│ checking vfork.h usability... no
│ checking vfork.h presence... no
│ checking for vfork.h... no
│ checking for fork... yes
│ checking for vfork... yes
│ checking for working fork... yes
│ checking for working vfork... (cached) yes
│ checking for popen... yes
│ checking for posix_spawn... yes
│ checking for sig_atomic_t... yes
│ checking for signal... yes
│ checking for sigaction... yes
│ checking for setpgid... yes
│ checking for library containing dlopen... none required
│ checking sys/time.h usability... yes
│ checking sys/time.h presence... yes
│ checking for sys/time.h... yes
│ checking for getrusage... yes
│ checking for gettimeofday... yes
│ checking for clock_gettime... yes
│ checking for clock_getres... yes
│ checking for setitimer... yes
│ checking for vm_allocate... yes
│ checking for sbrk... yes
│ checking for madvise... yes
│ checking for sysconf... yes
│ checking for strlcpy... yes
│ checking for strlcmp... no
│ checking for strlcat... yes
│ checking for special C compiler options needed for large files... no
│ checking for _FILE_OFFSET_BITS value needed for large files... no
│ checking for library containing cos... none required
│ checking for log2... yes
│ checking for log10... yes
│ checking for log1p... yes
│ checking for exp2... yes
│ checking for expm1... yes
│ checking for exp10... no
│ checking for trunc... yes
│ checking for sigsetjmp... yes
│ checking for _setjmp... no
│ checking execinfo.h usability... yes
│ checking execinfo.h presence... yes
│ checking for execinfo.h... yes
│ checking size parameter type for backtrace()... int
│ checking for library containing backtrace... none required
│ configure: creating ./config.status
│ config.status: creating bin/gap.sh
│ config.status: creating GNUmakefile
│ config.status: creating doc/make_doc
│ config.status: creating gac
│ config.status: creating gen/config.h
│ config.status: executing gen/stamp-h commands
│ config.status: executing libtool commands
│ config.status: executing src/compiled.h commands
│ GAP_VERSION = 4.11.0
│ running autoconf
│ autom4te: cannot open configure: Permission denied
│ make: *** [/Users/mo/.julia/artifacts/5f84ea130b97f5f163682f0f9bac5ebff45e7d18/gap-4.11.0/configure] Error 1
│ [ Info: Downloading https://github.com/bicycle1885/ZlibBuilder/releases/download/v1.0.4/Zlib.v1.2.11.x86_64-apple-darwin14.tar.gz to /Users/mo/.julia/dev/GAP/deps/usr/downloads/Zlib.v1.2.11.x86_64-apple-darwin14.tar.gz...
│ [ Info: Downloading https://github.com/JuliaPackaging/Yggdrasil/releases/download/GMP-v6.1.2-1/GMP.v6.1.2.x86_64-apple-darwin14.tar.gz to /Users/mo/.julia/dev/GAP/deps/usr/downloads/GMP.v6.1.2.x86_64-apple-darwin14.tar.gz...
│ [ Info: Downloading https://github.com/benlorenz/ncursesBuilder/releases/download/v6.1/ncurses.v6.1.0.x86_64-apple-darwin14.tar.gz to /Users/mo/.julia/dev/GAP/deps/usr/downloads/ncurses.v6.1.0.x86_64-apple-darwin14.tar.gz...
│ [ Info: Downloading https://github.com/benlorenz/readlineBuilder/releases/download/v8.0/readline.v8.0.0.x86_64-apple-darwin14.tar.gz to /Users/mo/.julia/dev/GAP/deps/usr/downloads/readline.v8.0.0.x86_64-apple-darwin14.tar.gz...
│ ┌ Info: Compiling GAP with
│ │   gap_src_root = "/Users/mo/.julia/artifacts/5f84ea130b97f5f163682f0f9bac5ebff45e7d18/gap-4.11.0"
│ │   extra_gap_root = "/Users/mo/.julia/dev/GAP/"
│ └   gap_bin_root = "/Users/mo/.julia/dev/GAP/deps/build/gap-4.11.0-julia-1.4.0"
│ ERROR: LoadError: failed process: Process(`make -j16`, ProcessExited(2)) [2]
│
│ Stacktrace:
│  [1] pipeline_error at ./process.jl:525 [inlined]
│  [2] run(::Cmd; wait::Bool) at ./process.jl:440
│  [3] run at ./process.jl:438 [inlined]
│  [4] (::var"#3#4")() at /Users/mo/.julia/dev/GAP/deps/build.jl:62
│  [5] cd(::var"#3#4", ::String) at ./file.jl:104
│  [6] top-level scope at /Users/mo/.julia/dev/GAP/deps/build.jl:50
│  [7] include(::String) at ./client.jl:439
│  [8] top-level scope at none:5
│ in expression starting at /Users/mo/.julia/dev/GAP/deps/build.jl:50
[00:16:39] ######################################################################## 100.0%-#O#- #   #
[00:16:41] ######################################################################## 100.0%-#O#- #   #
[00:16:45] ######################################################################## 100.0%-#O#- #   #
[00:16:47] ######################################################################## 100.0%-#O#- #   #
│
│ Full log at /Users/mo/.julia/dev/GAP/deps/build.log
└ @ Pkg.Operations /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.4/Pkg/src/Operations.jl:892
```

Unfortunately, I still get the same error message:

```
│ GAP_VERSION = 4.11.0
│ running autoconf
│ autom4te: cannot open configure: Permission denied
│ make: *** [/Users/mo/.julia/artifacts/5f84ea130b97f5f163682f0f9bac5ebff45e7d18/gap-4.11.0/configure] Error 1
```

--- a/Artifacts.toml
+++ b/Artifacts.toml
@@ -1,6 +1,6 @@
 [gap-4-11-0]
-git-tree-sha1 = "5f84ea130b97f5f163682f0f9bac5ebff45e7d18"
+git-tree-sha1 = "848f0f318405266896d24d11e4baf5cf7bf3f203"
 
     [[gap-4-11-0.download]]
-    sha256 = "fe14498397fc9c13c9e55edda32b4c38a963ea4eb706780867c1f55748b678f3"
+    sha256 = "8bb16ff9fc48c260be38c0953b3321df1d6d42ad1f71e0999d379497cca44c05"
     url = "https://github.com/oscar-system/GAP.jl/releases/download/v0.3.5/gap-4.11.0.tar.bz2"

--- a/deps/build.jl
+++ b/deps/build.jl
@@ -56,6 +56,7 @@ cd(gap_bin_root) do
                 --with-gmp=$(deps_prefix)
                 --with-readline=$(deps_prefix)
                 --with-zlib=$(deps_prefix)
+                --disable-maintainer-mode
                 LIBS="-Wl,-rpath -Wl,$(deps_prefix)/lib"
                 ARCHEXT=v$(VERSION)
                 `)

